modules/rank-eval/src/test/resources/rest-api-spec/test/rank_eval/30_template.yaml

```diff
@@ -0,0 +1,68 @@
+---
+"Template request":
+  - do:
+     indices.create:
+         index: foo
+         body:
+           settings:
+             index:
+               number_of_shards: 1
+  - do:
+      index:
+        index:   foo
+        type:    bar
+        id:      doc1
+        body:    { "text": "berlin" }
+
+  - do:
+      index:
+        index:  foo
+        type:   bar
+        id:     doc2
+        body:   { "text": "amsterdam" }
+
+  - do:
+      index:
+        index:  foo
+        type:   bar
+        id:     doc3
+        body:   { "text": "amsterdam" }
+        
+  - do:
+      index:
+        index:  foo
+        type:   bar
+        id:     doc4
+        body:   { "text": "something about amsterdam and berlin" }
+
+  - do:
+      indices.refresh: {}
+
+  - do:
+      rank_eval_template:
+        body: {
+          "spec_id" : "cities_qa_queries",
+          "template": "{ \"query\": { \"match\" : {\"text\" : \"{{query_string}}\" }}}",
+          "requests" : [
+            {
+                "id": "amsterdam_query",
+                "params": { "query_string": "amsterdam_1" },
```

Comment:
Why do you use `amsterdam_1` and `berlin_1` in the query, when the docs contain `amsterdam` and `berlin`?